src/asmcnc/comms/model_manager.py

```diff
@@ -160,6 +160,31 @@ def is_machine_drywall(self):
         """
         return self._data['product_code'] is ProductCodes.DRYWALLTEC
 
+    def is_machine_sb(self):
+        # () -> bool
+        """
+        Returns True if the machine is a SmartBench machine, False otherwise.
+        """
+        return self._data['product_code'] in [
+            ProductCodes.PRECISION_PRO_X,
+            ProductCodes.PRECISION_PRO_PLUS,
+            ProductCodes.PRECISION_PRO,
+            ProductCodes.STANDARD,
+            ProductCodes.FIRST_VERSION
+        ]
+
+    def get_machine_type(self):
```

Comment:
Maybe make an enum for type, makes it easier to work with in the future if you don't need to know the exact string